### DISCUSSION TITLE
About default language

### DISCUSSION THREAD

**Discussion by hehetiger** [abpframework/abp#21556] [Q&A]
In multilingual files, I have only configured Chinese and English. When switching to Japanese, English will be displayed by default. How can I set it to display Chinese by default if the resource file for a certain language is not configured? I tried this method https://github.com/abpframework/abp/discussions/11873 , but it didn't work
![企业微信截图_20241206095139](https://github.com/user-attachments/assets/cba1f8e2-004a-4f44-9075-eaf39774214f)
![企业微信截图_20241206095159](https://github.com/user-attachments/assets/7aea4ba6-90a7-48be-b9d5-8c3c5eda5a6e)

**Comment by maliming** [comment]
Duplicate https://github.com/aspnetboilerplate/aspnetboilerplate/issues/7073
